### PR TITLE
Feature/banner

### PR DESCRIPTION
**Pull request by ghost**
This pull request adds the banner section. It's slightly different from the original. Some of that is because I wanted to make it look nicer on devices with relatively short heights (like mobile device landscape modes), which the original site doesn't seem to do well.

I had to add a JavaScript polyfill for the smooth scrolling so that browsers without that feature (Edge and Safari) will still be able to see the smooth scrolling affect. At some point in the future, we'll want to replace that code with jQuery so it's more customizable and closer to the original.

### CODE REVIEW

**Review by shalinidc — APPROVED** [review]
Looks good to me @camper-fcc. We can go ahead and merge this 👍

**Comment by ghost** [comment]
@shalinidc Great, thanks! :sunny:

I'll wait to merge this until after your footer code is merged so you don't have to fix merge conflicts.

**Pull request merged**
